src/Setup/DevDivInsertionFiles/DevDivInsertionFiles.sln

```diff
@@ -5,10 +5,6 @@ VisualStudioVersion = 15.0.26405.2
 MinimumVisualStudioVersion = 10.0.40219.1
 Project("{778DAE3C-4631-46EA-AA77-85C1314464D9}") = "DevDivInsertionFiles", "DevDivInsertionFiles.vbproj", "{5FE910A9-E7C9-40E1-AAF1-6C6C7DD61B67}"
 EndProject
-Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "DevDivPackagesRoslyn", "..\DevDivPackages\Roslyn\DevDivPackagesRoslyn.csproj", "{57557F32-0635-421A-BCF8-549AED50F764}"
-EndProject
-Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "DevDivPackagesDebugger", "..\DevDivPackages\Debugger\DevDivPackagesDebugger.csproj", "{18BDB5D5-340B-4834-B4FC-0327729A4A26}"
-EndProject
```

Comment:
Why are you removing DevdiVPackagesDebugger? Had that in this solution to make the inner loop here easy:

- Restore DevDivInsertionFiles.sln
- F5

Reply:
This PR changes your loop to restore Roslyn.sln, F5. That's not changing either way, so no reason to have it here.

Reply:
(and yes the restore is more expensive there. But now it means compiler dependencies are actually being considered, so the next time somebody from your team adds a dependency there's a chance the right thing will happen, instead of nothing happening)

Reply:
DevDivInsertionFiles.vbproj is not in Roslyn.sln. It would need to be added there to keep the loop you subscribe. I'm fine with doing that but probably should delete DevDivInsertionFiles.sln at the same time. It's only purpose was to support that specific developer loop.

Reply:
> It's only purpose was to support that specific developer loop.

Well, when DevDivInsertionFiles.sln was created, Roslyn wasn't even using NuGet -- we were just checking in references into our TFVC repo... :smile:

The restore inputs of DevDivInsertionFiles.vb isn't needed as an input in anything anyways -- F5 would probably still do it. I'm happy to delete the solution file and move the project in if you'd like.

Reply:
> The restore inputs of DevDivInsertionFiles.vb isn't needed as an input in anything anyways 

Sure but the restore of what used to be DevDivInsertionFiles.sln was a nice inner dev loop and a functional requirement in our old setup. Given the deletion of the project references the SLN file can't be restored + F5'd and the project itself doesn't need to be restored hence the SLN is pointless now.

>  I'm happy to delete the solution file and move the project in if you'd like.

I'd prefer that. It was my eventual plan anyways.

Reply:
I guess the functional requirement was undocumented, and perhaps only known by you. 😉 

Will do the deletion, but going to do that in a separate PR. @tmat wants this in to facilitate the System.Collections.Immutable cleanup (as he'll hit the same things I did without this), and I don't want to delay this one much further than necessary.

Reply:
> I guess the functional requirement was undocumented, and perhaps only known by you. 

And the test infrastructure that I wrote to make sure it worked 😄 

> Will do the deletion, but going to do that in a separate PR.

Okay